src/components/NftRewards/NftVideo/JuiceVideoThumbnailOrImage.tsx

```diff
@@ -3,12 +3,13 @@ import { ImageProps } from 'antd'
 import { JuiceVideoOrImgPreview } from 'components/Create/components/JuiceVideoOrImgPreview'
 import { useContentType } from 'hooks/ContentType'
 import { useState } from 'react'
+import { stopPropagation } from 'react-stop-propagation'
 import { classNames } from 'utils/classNames'
 import { fileTypeIsVideo } from 'utils/nftRewards'
 import { JuiceVideoThumbnail, PlayIconPosition } from './JuiceVideoThumbnail'
 
 export function JuiceVideoThumbnailOrImage({
-  isSelected,
+  isSelected = true,
```

Comment:
**question** why default this to true? Generally better to make props false by default, then enable where needed.

Reply:
If it defaulted to `false` then images/video thumbnails would be darkened by default. We could change the prop to something like `darkened` but it's less clear in the selectable instances.